ui/lib/core/addon/utils/client-count-utils.js

```diff
@@ -108,7 +108,7 @@ export const namespaceArrayToObject = (totalClientsByNamespace, newClientsByName
     const newNamespaceCounts = newClientsByNamespace?.find((n) => n.label === ns.label);
     if (newNamespaceCounts) {
       const { label, clients, entity_clients, non_entity_clients } = newNamespaceCounts;
-      const newClientsByMount = [...newNamespaceCounts?.mounts];
+      const newClientsByMount = [...newNamespaceCounts.mounts];
```

Comment:
It doesn't like the optional chaining here and since we are accessing `newNamespaceCounts` on the line above it shouldn't be necessary since an error would be thrown there if undefined.